I have no idea why, but this update seems to break a single test of `wandb`:

```
=================================== FAILURES ===================================
__________________________________ test_dcgm ___________________________________
[gw6] linux -- Python 3.11.9 /nix/store/zy22nlap6gznpjhqry4w0hgw5mf62q3f-python3-3.11.9/bin/python3.11

test_settings = <function test_settings.<locals>.update_test_settings at 0x7ffe829840e0>

    def test_dcgm(test_settings):
        with mock.patch.object(
            wandb.sdk.internal.system.assets.open_metrics.requests.Session,
            "get",
            mocked_requests_get,
        ):
            interface = AssetInterface()
            settings = SettingsStatic(
                test_settings(
                    dict(
                        _stats_sample_rate_seconds=1,
                        _stats_samples_to_average=1,
                    )
                ).to_proto()
            )
            shutdown_event = threading.Event()

            url = "http://localhost:9400/metrics"

            dcgm = OpenMetrics(
                interface=interface,
                settings=settings,
                shutdown_event=shutdown_event,
                name="dcgm",
                url=url,
            )

>           assert dcgm.is_available(url)
E           AssertionError: assert False
E            +  where False = is_available('http://localhost:9400/metrics')
E            +    where is_available = <wandb.sdk.internal.system.assets.open_metrics.OpenMetrics object at 0x7ffe8299a810>.is_available

/build/source/tests/pytest_tests/unit_tests/test_system_metrics/test_open_metrics.py:147: AssertionError
```

--- a/pkgs/servers/mlflow-server/default.nix
+++ b/pkgs/servers/mlflow-server/default.nix
@@ -1,9 +1,9 @@
-{ python3Packages, writers}:
+{ python3Packages, writers }:
 
 let
   py = python3Packages;
 
-  gunicornScript = writers.writePython3 "gunicornMlflow" {} ''
+  gunicornScript = writers.writePython3 "gunicornMlflow" { } ''
     import re
     import sys
     from gunicorn.app.wsgiapp import run
@@ -12,25 +12,28 @@ let
         sys.exit(run())
   '';
 in
-py.toPythonApplication
-  (py.mlflow.overridePythonAttrs(old: {
+py.toPythonApplication (
+  py.mlflow.overridePythonAttrs (old: {
 
     propagatedBuildInputs = old.dependencies ++ [
       py.boto3
       py.mysqlclient
     ];
 
-    postPatch = (old.postPatch or "") + ''
-      cat mlflow/utils/process.py
+    postPatch =
+      (old.postPatch or "")
+      + ''
+        cat mlflow/utils/process.py
 
-      substituteInPlace mlflow/utils/process.py --replace-fail \
-        "process = subprocess.Popen(" \
-        "cmd[0]='${gunicornScript}'; process = subprocess.Popen("
-    '';
+        substituteInPlace mlflow/utils/process.py --replace-fail \
+          "process = subprocess.Popen(" \
+          "cmd[0]='${gunicornScript}'; process = subprocess.Popen("
+      '';
 
     postInstall = ''
       gpath=$out/bin/gunicornMlflow
       cp ${gunicornScript} $gpath
       chmod 555 $gpath
     '';
-}))
+  })
+)

--- a/pkgs/servers/mlflow-server/default.nix
+++ b/pkgs/servers/mlflow-server/default.nix
@@ -1,32 +1,32 @@
-{ python3, writeText}:
+{ python3Packages, writers}:
 
 let
-  py = python3.pkgs;
+  py = python3Packages;
+
+  gunicornScript = writers.writePython3 "gunicornMlflow" {} ''
+    import re
+    import sys
+    from gunicorn.app.wsgiapp import run
+    if __name__ == '__main__':
+        sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', ''', sys.argv[0])
+        sys.exit(run())
+  '';
 in
 py.toPythonApplication
-  (py.mlflow.overridePythonAttrs(old: rec {
+  (py.mlflow.overridePythonAttrs(old: {
 
-    propagatedBuildInputs = old.propagatedBuildInputs ++ [
+    propagatedBuildInputs = old.dependencies ++ [
       py.boto3
       py.mysqlclient
     ];
 
     postPatch = (old.postPatch or "") + ''
-      substituteInPlace mlflow/utils/process.py --replace \
-        "child = subprocess.Popen(cmd, env=cmd_env, cwd=cwd, universal_newlines=True," \
-        "cmd[0]='$out/bin/gunicornMlflow'; child = subprocess.Popen(cmd, env=cmd_env, cwd=cwd, universal_newlines=True,"
-    '';
+      cat mlflow/utils/process.py
 
-    gunicornScript = writeText "gunicornMlflow"
-    ''
-        #!/usr/bin/env python
-        import re
-        import sys
-        from gunicorn.app.wsgiapp import run
-        if __name__ == '__main__':
-          sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', ''', sys.argv[0])
-          sys.exit(run())
-      '';
+      substituteInPlace mlflow/utils/process.py --replace-fail \
+        "process = subprocess.Popen(" \
+        "cmd[0]='${gunicornScript}'; process = subprocess.Popen("
+    '';
 
     postInstall = ''
       gpath=$out/bin/gunicornMlflow

